.github/workflows/check.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           files: |
             ${{ inputs.resource_mapping_directory }}/**/*.{yaml,yml}
       - name: 'Copy added and changed files'
-        if: ${{ steps.changed-yaml-files.outputs.any_changed == 'true' }}
+        if: '${{ steps.changed-yaml-files.outputs.any_changed == ''true'' }}'
```

Comment:
@sethvargo Hi, with this, vscode IDE gives an error even though workflows do not complain, wondering what setting in the IDE can fix it.
```
Unexpected symbol: 'true'''. Located at position 30 within expression: needs.check.outputs.aod == ''true''
```

Reply:
Hey @sqin2019 - hmm that feels like a buge in the editor. Which plugin is throwing the error?

Reply:
Not sure, will check, so no special IDE settings are needed for this I assume.

Reply:
seems like it is the github action extention https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions

Reply:
Got it - it feels like an extension of https://github.com/github/vscode-github-actions/issues/60? Do you mind filing an issue there?

Reply:
https://github.com/github/vscode-github-actions/issues/211